api/bolt/migrator/migrate_dbversion12.go

```diff
@@ -0,0 +1,20 @@
+package migrator
+
+import "github.com/portainer/portainer"
+
+func (m *Migrator) updateSettingsToVersion13() error {
+	legacySettings, err := m.settingsService.Settings()
+	if err != nil {
+		return err
+	}
+
+	legacySettings.AuthenticationMethod = portainer.AuthenticationInternal
```

Comment:
Question: why is it needed to set authentication method to internal in the migration?

Reply:
Definitely not needed. Copy/paste error. Will remove.